integration/grafana/README.md

```diff
@@ -23,3 +23,83 @@ is the `geopmexporter(1)`
 file - `GEOPM_Report.grafana.dashboard.json`, in this directory, 
 relies on `geopmexporter(1)` for its data source.
 
+
+## Monitoring PBS jobs
+
+The `geopmexporter(1)` is typically deployed system wide with either a systemd
+service or as a kubernetes service.  This documentation is for an alternative
+`geopmexporter(1)` deployment scenario where an unprivileged user of a PBS
+managed system would like to monitor the compute resources granted to the user
+only while they are allocated.  This methodology should not be applied for
+system-wide monitoring by an administrator, but rather for job monitoring by an
+end user of a PBS system where the geopmexporter is not deployed system-wide.
+
+A script is provided called `pbs_prometheus_launch.sh` that can be used to
+monitor jobs submitted to a PBS batch system.  The script runs the prometheus
+and grafana server on the head node of a system, and will connect to
+`geopmexporter(1)` clients running on the compute nodes of a user's job.
+
+### Requirements
+
+A user installation of grafana and prometheus is required.  We recommend
```

Comment:
Both grafana and prometheus are aviailable in spack.

There should be a section that discusses how to set things up that way as well.  At the very least a placeholder section that will be populated when we do the next release.